internal/plugins/capacity_liquid.go

```diff
@@ -90,6 +94,19 @@ func (p *liquidCapacityPlugin) Scrape(ctx context.Context, backchannel core.Capa
 		logg.Fatal("ServiceInfo version for %s changed from %d to %d; restarting now to reload ServiceInfo...",
 			p.LiquidServiceType, p.LiquidServiceInfo.Version, resp.InfoVersion)
 	}
+	resourceNames := SortMapKeys(p.LiquidServiceInfo.Resources)
+	var errs []error
+	for _, resourceName := range resourceNames {
+		perAZ := resp.Resources[resourceName].PerAZ
+		topology := p.LiquidServiceInfo.Resources[resourceName].Topology
+		err := MatchLiquidReportToTopology(perAZ, topology)
+		if err != nil {
+			errs = append(errs, fmt.Errorf("service: %s, resource: %s: %w", p.ServiceType, resourceName, err))
```

Comment:
This does not need to specify the service type, since the caller already adds that to the error message:
```suggestion
			errs = append(errs, fmt.Errorf("resource: %s: %w", resourceName, err))
```
Same in the QuotaPlugin below.

Reply:
I made a separate commit for it where I adjusted both.